aws/resource_aws_ec2_transit_gateway_route.go

```diff
@@ -92,6 +92,10 @@ func resourceAwsEc2TransitGatewayRouteRead(d *schema.ResourceData, meta interfac
 		return nil
 	})
 
+	if isResourceTimeoutError(err) {
+		_, err = ec2DescribeTransitGatewayRoute(conn, transitGatewayRouteTableID, destination)
```

Comment:
In this case we should also be assigning `transitGatewayRoute` so if it is found, it is properly handled by the rest of the `Read` function 👍 

```suggestion
		transitGatewayRoute, err = ec2DescribeTransitGatewayRoute(conn, transitGatewayRouteTableID, destination)
```